wp-content/themes/reconasia/assets/_scss/layout/_footer.scss

```diff
@@ -60,31 +72,36 @@
   &__logo {
     grid-area: logo;
     max-width: 300px;
```

Comment:
Since this isn't actually making the logo larger, I would update the width and height in the SVG itself.  You can find it under `/assets/static/csis-logo.svg`.  You'll need to update the width and height to get the SVG to actually display at 280.